doc/installationguide.md

```diff
@@ -236,6 +236,8 @@ used for the same purpose. For instance:
     compute expressions, possible values are: `legacy` or `jexl`. When not set or 
```

Comment:
With regards to documentation, having a look to the API description document, if my understanding is correct, I think the new field needs to be added to the following tables:

- https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#service-group-model
- https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#device-model

Reply:
> With regards to documentation, having a look to the API description document, if my understanding is correct, I think the new field needs to be added to the following tables:
> 
> * https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#service-group-model
> * https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#device-model

@fgalan We need to add details of new field "explicitAttrs" in both document as mentioned by you. Please confirm my understanding.

Reply:
That's correct. Although actually they are two sections/tables in the same document (api.md), rather than two documents.

Reply:
> With regards to documentation, having a look to the API description document, if my understanding is correct, I think the new field needs to be added to the following tables:
> 
> * https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#service-group-model
> * https://github.com/telefonicaid/iotagent-node-lib/blob/master/doc/api.md#device-model

@fgalan Hi, I have been incorporated the changes suggested by you. Thanks!